### ISSUE TITLE
disabled  b-drop-down-item accessibility issue

### ISSUE THREAD

**Issue by adholland** [bootstrap-vue-next/bootstrap-vue-next#1350]
version 0.10.1

b-drop-down-item marked as disabled currently fails accessibility tests. The issue is due to the disabled element requiring the associated aria-disabled attribute set to true. This is with the href set, generating a link, not sure without.

So for the following: -
```
<b-dropdown-item href="#" class="file-size-desc" :disabled="selectedSort === 'fileSizeDesc'" @click="handleSort('fileSize', false)">
        Name Descending
</b-dropdown-item>
```

produces: -
`<li data-v-d0da8fdb="" role="presentation"><a class="dropdown-item file-name-desc" disabled="true" href="#" target="_self">Name Descending</a></li>`

it should be : -
`<li data-v-d0da8fdb="" role="presentation"><a class="dropdown-item file-name-desc" disabled="true" aria-disabled="true" href="#" target="_self">Name Descending</a></li>`

This looks like it would be a straight forward fix, so would be great if this could be resolved soon, as working round it has become a real pain. 

Thanks
